templates/copy-template/readmes/README.md

```diff
@@ -19,7 +18,16 @@ A bundle of predefined templates to use can be found in [Common Templates](https
 
 ### Usage
 
-Please see [examples](examples) on how to copy a template.
+Please see examples on how to copy a template:
+Task run:
+```
+{{ task_run_yaml | to_nice_yaml }}
```

Comment:
What is expected to users to apply here? A yaml manifest like the previous ones? If that the case, please add it here or reference it?

Reply:
Please, check also the rest of these examples.

Reply:
I don't understand your comment. This is a jinja template which will be populated with variables by ansible during generating.